lib/de/ser/ializer.rb

```diff
@@ -59,6 +59,8 @@ def parse_field(object, field, value)
 
           return if parsed_value.nil?
 
+          return public_send(field.setter, object, parsed_value) if respond_to?(field.setter)
```

Comment:
Could you get away with a [custom attribute type](https://github.com/jsteinberg/ializer#registering-custom-attribute-types)?

Alternatively, I dont like `respond_to?` very much.  Maybe this should be the base behavior and we can define a generic setter method for each field ([like we do for serializers](https://github.com/jsteinberg/ializer/blob/7c06d0157eed4a1f0cc2a86a1c0e066209cfd9ff/lib/ser/ializer.rb#L139-L143))